app/views/play/level/PlayLevelView.coffee

```diff
@@ -733,8 +733,9 @@ module.exports = class PlayLevelView extends RootView
       contactModal.updateScreenshot?()
 
   onSurfaceContextMenu: (e) ->
-    return unless @surface.showCoordinates and ( navigator.clipboard or document.queryCommandSupported('copy') )
     e?.preventDefault?()
```

Comment:
why move this line before the return?

Reply:
I don't want either context menu to happen when clicking the Surface.